aea/helpers/base.py

```diff
@@ -119,6 +120,42 @@ def locate(path):
     return object
 
 
+def load_aea_package(configuration: ComponentConfiguration) -> None:
+    """
+    Load the AEA package.
+
+    It adds all the __init__.py modules into `sys.modules`.
+
+    :param configuration: the configuration object.
+    :return: None
+    """
+    dir = configuration.directory
+    assert dir is not None
+
+    # patch sys.modules with dummy modules
+    prefix_root = "packages"
+    prefix_author = prefix_root + f".{configuration.author}"
+    prefix_pkg_type = prefix_author + f".{configuration.component_type.to_plural()}"
+    prefix_pkg = prefix_pkg_type + f".{configuration.name}"
+    sys.modules[prefix_root] = types.ModuleType(prefix_root)
+    sys.modules[prefix_author] = types.ModuleType(prefix_author)
+    sys.modules[prefix_pkg_type] = types.ModuleType(prefix_pkg_type)
```

Comment:
basically, for each prefix of the import path we add a dummy module in `sys.modules`, e.g.
```
sys.modules["packages"] = dummy
sys.modules["packages.fetchai"] = dummy
sys.modules["packages.fetchai.protocols"] = dummy
```
So import statements that use these as prefix of the import path will not fail.